deluge/ui/gtk3/glade/main_window.ui

```diff
@@ -199,9 +197,7 @@
                         <property name="use_underline">True</property>
                         <property name="image">prefs-image</property>
                         <property name="use_stock">False</property>
-                        <property name="accel_group">accelgroup1</property>
                         <signal name="activate" handler="on_menuitem_preferences_activate" swapped="no"/>
-                        <accelerator key="P" signal="activate" modifiers="GDK_CONTROL_MASK"/>
```

Comment:
I would prefer if these accelerator keys were not removed from the glade files and instead we found an alternative way to do this.

Reply:
I don't think it's possible to implement portable shortcuts without removing them from the glade files.